playwright.config.ts

```diff
@@ -0,0 +1,107 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+	testDir: './src/ui/tests',
```

Comment:
Can we move these tests to `src/test/ui` so all our testing is one place?

Reply:
for sure, will move